apps/files/src/components/SidebarQuota.vue

```diff
@@ -0,0 +1,162 @@
+<!--
+SPDX-FileCopyrightText: 2023 Nextcloud GmbH and Nextcloud contributors
+SPDX-FileCopyrightText: 2024 STRATO AG
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<NcAppNavigationItem :aria-label="t('files', 'Storage informations')"
+		:class="{ 'app-navigation-entry__settings-quota--not-unlimited': storageStats.quota >= 0}"
+		:loading="loadingStorageStats"
+		:name="storageStatsTitle"
+		:title="storageStatsTitle + ': ' + storageStatsTextUsed + ' ' + storageStatsTextOf"
+		class="app-navigation-entry__settings-quota"
+		data-cy-files-navigation-settings-quota
+		@click.stop.prevent="debounceUpdateStorageStats">
+		<CloudIcon slot="icon" :size="16" />
+		<NcProgressBar v-if="storageStats.quota >= 0"
+			slot="extra"
+			size="7"
+			:error="storageStats.relative > 80"
+			:value="Math.min(storageStats.relative, 100)"
+			:color="barColor"
+			:style="{'background-color': backgroundColor}" />
+		<div slot="extra" class="quota-text">
+			<span class="quota-text-bold">{{ storageStatsTextUsed }}</span>
+			<span>{{ storageStatsTextOf }}</span>
+		</div>
+	</NcAppNavigationItem>
+</template>
+
+<script>
+import { debounce, throttle } from 'throttle-debounce'
+import { formatFileSize } from '@nextcloud/files'
+import { generateUrl } from '@nextcloud/router'
+import { loadState } from '@nextcloud/initial-state'
+import { subscribe } from '@nextcloud/event-bus'
+import { translate } from '@nextcloud/l10n'
+import axios from '@nextcloud/axios'
+
+import CloudIcon from 'vue-material-design-icons/Cloud.vue'
+import NcAppNavigationItem from '@nextcloud/vue/dist/Components/NcAppNavigationItem.js'
+import NcProgressBar from '@nextcloud/vue/dist/Components/NcProgressBar.js'
+
+export default {
+	name: 'SidebarQuota',
+
+	components: {
+		CloudIcon,
+		NcAppNavigationItem,
+		NcProgressBar,
+	},
+
+	data() {
+		return {
+			loadingStorageStats: false,
+			storageStats: loadState('files', 'storageStats', null),
+			barColor: null,
+			backgroundColor: null,
+		}
+	},
+
+	computed: {
+		storageStatsTitle() {
+			return this.t('files', 'Storage used')
+		},
+		storageStatsTextUsed() {
+			return this.t('files', '{used}', { used: formatFileSize(this.storageStats?.used, false, false) })
```

Comment:
todo: there is nothing to translate in this string
```suggestion
			return formatFileSize(this.storageStats?.used, false, false);
```